matcalc/eos.py

```diff
@@ -65,17 +66,15 @@ def calc(self, structure: Structure) -> dict:
         if self.relax_structure:
             relaxer = RelaxCalc(self.calculator, optimizer=self.optimizer, fmax=self.fmax, steps=self.steps)
             structure = relaxer.calc(structure)["final_structure"]
-
+        adaptor = AseAtomsAdaptor()
         volumes, energies = [], []
-        relaxer = RelaxCalc(
-            self.calculator, optimizer=self.optimizer, fmax=self.fmax, steps=self.steps, relax_cell=False
-        )
         for idx in np.linspace(-self.max_abs_strain, self.max_abs_strain, self.n_points):
             structure_strained = structure.copy()
             structure_strained.apply_strain([idx, idx, idx])
-            result = relaxer.calc(structure_strained)
-            volumes.append(result["final_structure"].volume)
-            energies.append(result["energy"])
+            atoms = adaptor.get_atoms(structure_strained)
```

Comment:
This change means the atoms are no longer allowed to move? Are you sure this is what we want? Previously the cell was held fixed but the atoms relaxed for each strain-level. Perhaps @shyuep can comment.

Reply:
@janosh, Thanks a lot for reviewing the code. I just checked with a small example and I found that the apply_strain function in pymatgen allowing both lattices and atomic positions to change together. please correct me if I am wrong

Reply:
I wasn't referring to `apply_strain`. I meant using the M3GNet `relaxer` _after_ applying each strain allowed the atom positions to relax at fixed volume in each step which now is no longer the case. I'm just making sure this change is intentional and wondering about the reason?

Reply:
Hi @janosh, sorry for the late reply. I discussed with Shyue Ping and the relaxation is required for non-cubic crystal structures to perform EOS fitting. I think I will cancel the PR now and thanks for the discussion.